src/app/local-storage-manager.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Todo } from './interfaces/todo';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalStorageManagerService {
+
+  getLocalStorage(key) {
+    return JSON.parse(localStorage.getItem(key));
+  }
+
+  setLocalStorageTodos(todos: Todo[]): void {
```

Comment:
Again, this shouldn't be aware of what it's attempting to store, we should just have a generic get and set methods

Reply:
Generalised this method, users pass in a key(string) and the object.